### PR TITLE
feat(text): add support for line height

### PR DESCRIPTION
**Pull request by deorukhkarchaitanya**

### CODE REVIEW

File: src/_helpers/fonts.js [hunk 1]
```diff
@@ -1,5 +1,14 @@
...
-  return parseFloat(theme.fonts.lineHeight[lineHeight]) * parseFloat(theme.fonts.size[size]);
+const getDownwardEvenNumber = (val) => {
+  if (val % 2 === 0) {
+    return `${val}px`;
+  } else {
+    return getDownwardEvenNumber(val - 1);
+  }
+};
+
+const getLineHeight = (fontSize, lineHeightMultiplier) => {
```
Comment (kamleshchandnani) [inline src/_helpers/fonts.js]: So we have a 1.4 multiplier. we can get rid of all this 😆

Comment (deorukhkarchaitanya) [reply src/_helpers/fonts.js]: I can't see those changes on figma. 1.4 multiplier for everything?

File: src/_helpers/fonts.js [hunk 2]
```diff
@@ -0,0 +1,17 @@
+const getLineHeightFromTextSize = (textSize, theme) => {
```
Comment (kamleshchandnani) [inline src/_helpers/fonts.js]: rename to `getLineHeight`
Also rearrange the arguments in colors we have theme first and then the color let's follow the same here

Comment (deorukhkarchaitanya) [reply src/_helpers/fonts.js]: done

**Pull request merged**